thesis/experience.tex

```diff
@@ -0,0 +1,25 @@
+\chapter{Experience}
+In the process of developing the applications and working with the domain we have made some valuable experiences.
+
+Overall, the system engineering challenge of grasping the complete domain, breaking it down into consumable parts and understanding the needs of each of these is a repeatable challenging task.
```

Comment:
repeatable challenging task?  repeatable?  i dont think i understand it.. maybe its perfectly fine :)